api_reference/classes/systemmanager.rst

```diff
@@ -511,6 +508,66 @@ to place the ports tree.
   "namespace": "sysadm"
  }
 
+.. index:: systemmanager getsysctl
+.. _Get System Ctl:
+
+Get System Ctl
+==============
+
+:command:`"getsysctl"` reports back the value of the requested
+sysctl(s).
+Required arguments: "sysctl" (string or array of strings).
+
+.. index:: systemmanager halt
+.. _Halt the System:
```

Comment:
systemmanager halt and halt the system index entry/internal link need to be moved down to their proper section.